server/test/write_authzmodel.go

```diff
@@ -532,6 +532,117 @@ func WriteAuthorizationModelTest(t *testing.T, datastore storage.OpenFGADatastor
 			},
 			err: errors.InvalidAuthorizationModelInput(typesystem.InvalidRelationError("repo", "viewer")),
 		},
+		{
```

Comment:
Can we add one more test for the following model

```
type folder
  relations
    define parent: [folder]
    define viewer as viewer from parent
```

Reply:
added!